packages/components/inputs/checkbox-input/src/checkbox-input.story.js

```diff
@@ -18,28 +27,80 @@ storiesOf('Components|Inputs', module)
   .add('CheckboxInput', () => (
     <Section>
       <Spacings.Stack>
-        <Value
-          defaultValue={false}
-          render={(isChecked, onChange) => (
-            <CheckboxInput
-              id={text('id', '')}
-              name={text('name', '')}
-              onChange={(event) => {
-                action('onChange')(event);
-                onChange(!isChecked);
-              }}
-              value={text('value', '')}
-              isChecked={isChecked}
-              isIndeterminate={boolean('isIndeterminate', false)}
-              isHovered={boolean('isHovered', false)}
-              isDisabled={boolean('isDisabled', false)}
-              isReadOnly={boolean('isReadOnly', false)}
-              hasError={boolean('hasError', false)}
-            >
-              {text('Label', 'This is a label')}
-            </CheckboxInput>
-          )}
-        />
+        {Object.entries(states).map(([stateKey, stateLabel]) => (
+          <Grid
+            key={stateKey}
+            alignItems="center"
+            gridTemplateColumns="100px 1fr 1fr 1fr"
+          >
+            <Text.Body isBold>{stateLabel}</Text.Body>
+            <Value
+              defaultValue={true}
+              render={(isChecked, onChange) => (
+                <CheckboxInput
+                  id={text('id', '')}
+                  name={text('name', '')}
+                  onChange={(event) => {
+                    action('onChange')(event);
+                    onChange(!isChecked);
+                  }}
+                  value={text('value', '')}
+                  isChecked={isChecked}
+                  isIndeterminate={boolean('isIndeterminate', false)}
+                  isHovered={boolean('isHovered', false)}
```

Comment:
Just wanted to note that unlike in the previous story, this knob doesn't seem to take any effect:
Previous:
![Feb-20-2023 09-41-50](https://user-images.githubusercontent.com/49066275/220055646-674a9726-31ff-4a4c-bca1-059452e6da03.gif)
Now:
![Feb-20-2023 09-41-59](https://user-images.githubusercontent.com/49066275/220055772-f7ae9506-9dbe-41df-bc6c-6756e81e278a.gif)

Reply:
Ah good catch, I forgot about that prop. I'll fix that. Thanks

Reply:
Fixed here: https://github.com/commercetools/ui-kit/pull/2424/commits/74a824a19e7178276a8706fcbdb429dc5b197456

Reply:
The `isHovered` works great now 🙌 But it seems there's a regression with normal hovering over the label.

Reply:
😭 Thanks for noticing, will take another look.

Reply:
Alright, this should fix that: https://github.com/commercetools/ui-kit/pull/2424/commits/09da31aca7309ad7d6d0f53439095c1667ed5d51